src/haplotypes/model.rs

```diff
@@ -98,15 +99,19 @@ impl Likelihood {
     ) -> LogProb {
         // TODO compute likelihood using neg_binom on the counts and dispersion
         // in the data and the fractions in the events.
-        // Later: use the cache to avoid redundant computations.
-        // event
-        //     .0
-        //     .iter()
-        //     .zip(data.kallisto_estimates.iter())
-        //     .map(|(fraction, estimate)| {
-        //         Self::neg_binom(estimate.count, fraction, estimate.dispersion)
-        //     });
-        todo!()
+        //Later: use the cache to avoid redundant computations.
+        event
+            .0
```

Comment:
This `.0`looks fishy to me. At least I don't understand why that is done here. Either comment unpack the event or iterate over all entries. Whatever is appropriate here.